README.md

```diff
@@ -213,3 +213,6 @@ To build the documentation, ensure that you are in the top-level
 
 	admin/build-doc
 
+## Reporting Issues
+
+To report an issue and view existing issues, please visit https://tracker.ceph.com/issues.
```

Comment:
I would simply put "https://tracker.ceph.com", which would guide users to the home page. From there, they can navigate to the tab of their choice.